docs/azure/sdk/authentication/index.md

```diff
@@ -8,57 +8,74 @@ ms.date: 08/02/2024
 
 # Authenticate .NET apps to Azure services using the Azure Identity library overview
 
-When an app needs to access an Azure resource, the app must be authenticated to Azure. This is true for all apps, whether deployed to Azure, deployed on-premises, or under development on a local developer workstation. This article describes the recommended approaches to authenticate an app to Azure when using the Azure SDK client libraries.
+Apps must authenticate to Azure in order to access Azure services and resources. This requirement applies whether the app is deployed to Azure, hosted on-premises, or running locally on a developer workstation. The sections ahead describe the recommended approaches to authenticate an app to Azure across different environments when using the Azure SDK client libraries.
```

Comment:
Since the Azure Identity library focuses solely on Entra-based authN, let's change the wording.
```suggestion
Apps can use the Azure Identity library to authenticate to Microsoft Entra ID, which allows the apps to access Azure services and resources. This authentication requirement applies whether the app is deployed to Azure, hosted on-premises, or running locally on a developer workstation. The sections ahead describe the recommended approaches to authenticate an app to Microsoft Entra ID across different environments when using the Azure SDK client libraries.
```